homeassistant/components/decora_wifi/light.py

```diff
@@ -171,3 +135,55 @@ def update(self) -> None:
             self._switch.refresh()
         except ValueError:
             _LOGGER.error("Failed to update myLeviton switch data")
+
+    @property
+    def device_info(self) -> DeviceInfo:
```

Comment:
I think you can make this a shorthand `_attr_device_info`

Reply:
Are you suggestion replacing this `@property` with
```
    @property
    def device_info(self) -> DeviceInfo:
        DeviceInfo( ... )
```
```
        def __init_ ....
            self._attr_device_info = DeviceInfo(
            ...
        )
```

Reply:
Yes! There is no more logic to create the device info and a lot of the variables you use is already present there so it's a bit cheaper to do it this way

Reply:
fixed in https://github.com/home-assistant/core/pull/104573/commits/8f627725205a88235b463427843d8c265abeb108